x-pack/plugins/security_solution/public/flyout/left/components/host_details.tsx

```diff
@@ -0,0 +1,289 @@
+              showActionTooltips
+              triggerId={SecurityCellActionsTrigger.DEFAULT}
+              field={{
+                name: 'user.name',
+                value: user,
+                type: 'keyword',
+              }}
+            >
+              {user}
+            </SecurityCellActions>
+          </EuiText>
+        ),
+      },
+      {
+        field: 'ip',
+        name: i18n.RELATED_ENTITIES_IP_COLUMN_TITLE,
+        render: (ips: string[]) => {
+          return (
+            <DefaultFieldRenderer
+              rowItems={ips}
+              attrName={'host.ip'}
+              idPrefix={''}
+              isDraggable={false}
+              render={(ip) => (ip != null ? <NetworkDetailsLink ip={ip} /> : getEmptyTagValue())}
+            />
+          );
+        },
+      },
+      ...(isPlatinumOrTrialLicense
+        ? [
+            {
+              field: 'risk',
+              name: (
+                <EuiToolTip content={USER_RISK_TOOLTIP}>
+                  <>
+                    {ENTITY_RISK_CLASSIFICATION(RiskScoreEntity.user)}{' '}
+                    <EuiIcon color="subdued" type="iInCircle" className="eui-alignTop" />
+                  </>
+                </EuiToolTip>
+              ),
+              truncateText: false,
+              mobileOptions: { show: true },
+              sortable: false,
+              render: (riskScore: RiskSeverity) => {
+                if (riskScore != null) {
+                  return <RiskScore severity={riskScore} />;
+                }
+                return getEmptyTagValue();
+              },
+            },
+          ]
+        : []),
+    ],
+    [isPlatinumOrTrialLicense]
+  );
+
+  const relatedUsersCount = useMemo(
+    () => (
+      <EuiFlexGroup alignItems="center" gutterSize="s">
+        <EuiFlexItem grow={false}>
+          <EuiIcon type="user" />
+        </EuiFlexItem>
+        <EuiFlexItem grow={false}>
+          <EuiTitle size="xxxs">
+            <EuiText>{`${i18n.RELATED_USERS_TITLE}: ${totalCount}`}</EuiText>
+          </EuiTitle>
+        </EuiFlexItem>
+      </EuiFlexGroup>
+    ),
+    [totalCount]
+  );
+
+  const pagination: {} = {
+    pageSize: 4,
+    showPerPageOptions: false,
+  };
+
+  return (
+    <>
+      <EuiTitle size="xs">
+        <h4>{i18n.HOSTS_TITLE}</h4>
+      </EuiTitle>
+      <EuiSpacer size="s" />
+      <EntityPanel
+        title={hostName}
+        iconType={'storage'}
+        expandable={true}
+        expanded={true}
+        headerContent={relatedUsersCount}
+        data-test-subj={HOST_DETAILS_TEST_ID}
+      >
+        <EuiTitle size="xxs">
+          <h5>{i18n.HOSTS_INFO_TITLE}</h5>
+        </EuiTitle>
+        <EuiSpacer size="s" />
+        <AnomalyTableProvider
+          criteriaFields={hostToCriteria(hostDetails)}
+          startDate={from}
+          endDate={to}
+          skip={isInitializing}
+        >
+          {({ isLoadingAnomaliesData, anomaliesData, jobNameById }) => (
```

Comment:
why this function and not just render the component?

Reply:
`HostOverview` takes some anomalies data props (line 228) that is provided by the `AnomaliesTableProvider`